test/libsolidity/syntaxTests/inlineAssembly/shadowing/name_clash_in_import.sol

```diff
@@ -15,4 +15,4 @@ contract B {
 }
 // ----
 // DeclarationError 3859: (b:105-106): This declaration shadows a declaration outside the inline assembly block.
-// DeclarationError 3859: (b:128-131): The prefix of this declaration conflicts with a declaration outside the inline assembly block.
```

Comment:
I am actually not sure what the purpose of this error was - maybe in preparation for path resolution in inline assembly.